src/components/IngredientCard/IngredientCard.scss

```diff
@@ -0,0 +1,53 @@
+.ingredient-card {
+    display: flex;
```

Comment:
We're using a lot of unnecessary `flex`.

You can do everything you've done while avoid a few flexes.

```scss
.ingredient-card {
    // display: flex;
    // flex-direction: column;
    width: 9em;
    height: 8em;
    border-radius: 0.6em;
    overflow: hidden;
    padding: 0.6em;
    box-shadow: 0 0.4em 0.8em 0 rgba(0,0,0,0.2);

    .ingredient-header {
        display: flex;
        justify-content: start;
        align-items: center;
        width: 100%;
        height: 40%;

        .ingredient-image {
            // display: inline-block;
            width: auto;
            height: 100%;
        }

        .ingredient-name {
            // display: flex;
            // flex-direction: column;
            // justify-content: center;
            color: white;
            font-size: 18px;
            text-transform: uppercase;
            // width: auto;
            // height: 100%;
            overflow: hidden;
            text-align: center;
            vertical-align: middle;
        }
    }

    .ingredient-body {
        display: flex;
        flex-direction: column;
        justify-content: flex-end;
        width: 100%;
        height: 60%;

        .ingredient-tags {
            display: inline-block;

            > * {
                margin: 0px 0.15em;
            }
        }
    }
}
```

Note: Will also need to change `ingredient-name` to a span for the above css. Which makes sense for inline ements. Should use span for inline elements.